content/courses/native-onchain-development/program-state-management.md

```diff
@@ -100,77 +97,81 @@ and deserialize the data as needed.
 
 ### Creating accounts
 
-Before we can update the data field of an account, we have to first create that
-account.
+Before we can update an account's data field, we have to create that account
+first.
 
-To create a new account within our program we must:
+To create that account, we must:
 
 1. Calculate the space and rent required for the account
 2. Have an address to assign the new account
 3. Invoke the system program to create the new account
 
-#### Space and rent
+### Space and rent
 
-Recall that storing data on the Solana network requires users to allocate rent
-in the form of lamports. The amount of rent required by a new account depends on
-the amount of space you would like allocated to that account. That means we need
-to know before creating the account how much space to allocate.
+Recall that storing data on the Solana network requires users to allocate
+lamports as rent. The rent required by a new account depends on the space you
+would like allocated to that account. That means we must know how much space to
+allocate before creating the account.
 
 Note that rent is more like a deposit. All the lamports allocated for rent can
-be fully refunded when an account is closed. Additionally, all new accounts are
+be fully refunded when an account is closed. Additionally, all new accounts must
 now required to be
-[rent-exempt](https://twitter.com/jacobvcreech/status/1524790032938287105),
-meaning lamports are not deducted from the account over time. An account is
-considered rent-exempt if it holds at least 2 years worth of rent. In other
-words, accounts are stored onchain permanently until the owner closes the
-account and withdraws the rent.
+[rent-exempt](https://x.com/jacobvcreech/status/1524790032938287105), meaning
+lamports are not deducted from the account over time. An account is considered
+rent-exempt if it holds at least two years' rent. In other words, accounts are
+stored onchain permanently until the owner closes the account and withdraws the
+rent.
 
 In our note-taking app example, the `NoteState` struct specifies three fields
 that need to be stored in an account: `title`, `body`, and `id`. To calculate
-the size the account needs to be, you would simply add up the size required to
-store the data in each field.
+the account size, you would add the size of each data field.
```

Comment:
We should be able to use INIT_SPACE for this now to get the size from `NoteState`. See CONTRIBUTING.md